testing_and_setup/compass/load_compass_env.bash

```diff
@@ -0,0 +1,36 @@
+version=0.1.2
+
+default_python=3.7
+
+# The rest of the script should not need to be modified
+if [[ $HOSTNAME = "cori"* ]] || [[ $HOSTNAME = "dtn"* ]]; then
+  base_path="/global/cfs/cdirs/acme/software/anaconda_envs/base"
+elif [[ $HOSTNAME = "acme1"* ]] || [[ $HOSTNAME = "aims4"* ]]; then
+  base_path="/usr/local/e3sm_unified/envs/base"
+elif [[ $HOSTNAME = "blueslogin"* ]]; then
+  base_path="/lcrc/soft/climate/e3sm-unified/base"
+elif [[ $HOSTNAME = "rhea"* ]]; then
+  base_path="/ccs/proj/cli900/sw/rhea/e3sm-unified/base"
+elif [[ $HOSTNAME = "cooley"* ]]; then
+  base_path="/lus/theta-fs0/projects/ccsm/acme/tools/e3sm-unified/base"
+elif [[ $HOSTNAME = "compy"* ]]; then
+  base_path="/share/apps/E3SM/conda_envs/base"
+elif [[ $HOSTNAME = "gr-fe"* ]] || [[ $HOSTNAME = "wf-fe"* ]]; then
```

Comment:
Wolf can be left out since it's decommissioned, as far as I know. Should Badger be included?

Reply:
Yep, absolutely (another artifact from a previous script).